client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -134,6 +136,66 @@ export const useCommandsArrayWpcom = ( {
 		displayNotice( __( 'SSH/SFTP password reset and copied to clipboard.' ) );
 	};
 
+	const getEdgeCacheStatus = async ( siteId: number ) => {
+		const response = await wpcom.req.get( {
+			path: `/sites/${ siteId }/hosting/edge-cache/active`,
+			apiNamespace: 'wpcom/v2',
+		} );
+
+		return response;
+	};
+
+	const clearEdgeCache = async ( siteId: number ) => {
+		try {
+			const response = await getEdgeCacheStatus( siteId );
+
+			if ( response ) {
+				// If global cache is active, purge the cache
+				await wpcom.req.post( {
+					path: `/sites/${ siteId }/hosting/edge-cache/purge`,
+					apiNamespace: 'wpcom/v2',
+				} );
+				displayNotice( __( 'Succesfully cleared cache.' ) );
+			} else {
+				// If global edge cache is not active, clear WordPress cache
+				dispatch( clearWordPressCache( siteId, 'Cache not active' ) );
+			}
+		} catch ( error ) {
+			displayNotice( __( 'Failed to clear cache.' ), 'is-error' );
+		}
+	};
+
+	const toggleEdgeCache = async ( siteId: number ) => {
+		const active = await getEdgeCacheStatus( siteId );
+
+		const toggleCacheMessage = active
+			? __( 'Disabling global edge cache…' )
```

Comment:
```suggestion
			? __( 'Disabling edge cache…' )
```

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/84819/commits/24d50ad0b5e4028ec5986603bbec46342d55cbe9